lib/rules/valid-graphql-wire-adapter-callback-parameters.js

```diff
@@ -25,11 +25,16 @@ module.exports = {
                 const { decorators } = node;
 
                 // Check that the @wire decorator is using graphql
-                const graphQlDecorator = decorators.find((decorator) => {
-                    return decorator.expression.arguments.find((argument) => {
-                        return argument.name === 'graphql';
+                let graphQlDecorator;
+                if (decorators !== undefined) {
+                    graphQlDecorator = decorators.find((decorator) => {
+                        if (decorator.expression && decorator.expression.arguments) {
+                            return decorator.expression.arguments.find((argument) => {
+                                return argument.name === 'graphql';
+                            });
+                        }
```

Comment:
```suggestion
                        if (decorator.expression && decorator.expression.arguments) {
                            return decorator.expression.arguments.find((argument) => {
                                return argument.name === 'graphql';
                            });
                        }
                        return false;
```

Makes things a bit clearer IMO.